app/controllers/welcome_controller.rb

```diff
@@ -4,7 +4,7 @@ def index
       return @image_results_list if @image_results_list
       if params[:search_tattoos]
          image_search = GoogleSearchResults.new(q: "tattoo #{ params[:search_tattoos] }", tbm: "isch", serp_api_key: ENV['SERP_API_KEY'])
-         @image_results_list = image_search.get_hash[:images_results][0..29]
+         @image_results_list = image_search.get_hash[:images_results][0..9]
```

Comment:
perhaps this should return `[0..8]` since this page displays 3 images per row? Might look a little more polished?